docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+# JupyterLab
+
+ - Even if you don't use Jupyter, you can still take this survey. Just indicate this fact in the first question and carry on as best as you can.
+ - Thank you - your participation guides Jupyter's roadmap toward your real-life use cases.
```

Comment:
Can we add a couple of sentences about what we plan to do with the survey data, and how privacy will be managed, and who will see the survey data?

Reply:
@jasongrout is this along the lines of what you were going for?

"The aggregate survey data itself will be openly shared with the Jupyter community when polling closes in mid-December. If you opt to provide your email for a user interview, it not be used for Jupyter's promotional purposes and it will not be shared with a 3rd party."
^ Figured we will run it for a month depending on how many responses we get?

"Thank you. Your participation guides Jupyter's roadmap toward your real-life use cases by quantifiably helping us prioritize the functionality that is important to our userbase."